.github/PULL_REQUEST_TEMPLATE/new_component.md

```diff
@@ -0,0 +1,69 @@
+<!--- Use this only if you've created new ACE component (manager, util, etc.). Please provide a name of your newly created ACE component in title of pull request. -->
+
+## Description
+<!--- Describe your changes in few sentences. If your pull request is related to an issue, please put `Closes #issueNumber` or `Related to #issueNumber` sentence here. -->
+
+## Motivation and Context
+<!--- Why is this change required? What problem does it solve? -->
+<!--- If it fixes an open issue, please link to the issue here. -->
```

Comment:
Related issue is mentioned in `Description` already. Do you want to keep in doubled?

Reply:
good catch!